ts/input/tex/ams/AmsMappings.ts

```diff
@@ -112,7 +107,7 @@ new sm.CommandMap('AMSmath-macros', {
  */
 new sm.EnvironmentMap('AMSmath-environment', ParseMethods.environment, {
   'eqnarray*':   ['EqnArray', null, false, true, 'rcl',
-                  '0 ' + TexConstant.Length.THICKMATHSPACE, '.5em'],
+                  '0 ' + MATHSPACE.thickmathspace + 'em', '.5em'],
```

Comment:
There is an `em()` function in `util/lengths.ts` that handles converting the number to ems (with fewer decimals).  it would be good to use that here, and below rather than just adding `'em'`.  I see five other occurrences below.